data/json/items/magazine/308.json

```diff
@@ -44,6 +44,23 @@
     "reliability": 9,
     "flags": [ "MAG_COMPACT" ]
   },
+  {
+    "id": "fal_makeshiftmag",
+    "type": "MAGAZINE",
+    "name": "FN FAL makeshift magazine",
+    "description": "An improvised 5-round straight single-stack box magazine consisting of little more than a bent sheet of steel held together by duct tape and hope, for use with FN FAL rifle.",
+    "weight": "80 g",
+    "volume": "250 ml",
```

Comment:
I don't think magazines just so happen to have volumes based on the old 250ml volume units 😛

Reply:
I can't think of any way to better estimate the volume, especially considering that this is a makeshift magazine and all stats would actually vary wildly depending on the maker.

Reply:
@tenmillimaster & I(Tenmilli is more experienced at it) have both used [ImageJ](https://imagej.nih.gov/ij/) on images of guns to find volume, technique of which is easily applicable to magazines(use gun with a known length, scale image based on length, use scale on magazine to find its volume as well). I'd make its volume equal to the magazine it's copying this way(with the lower capacity as a penalty for being makeshift, obv).

Reply:
Actually the makeshift magazine would have to have less volume, I think. It'd be thinner due to being single-stack (most copied magazines are double-stack), and also shorter, and you can't really make larger magazines due to them becoming way too unreliable to be at least somewhat usable.